src/ocean/text/convert/Float.d

```diff
@@ -49,10 +49,22 @@ private alias real NumType;
 
 private extern (C)
 {
-    real log10l (real x);
-    real ceill (real num);
-    real modfl (real num, real *i);
-    real powl  (real base, real exp);
+    version (Windows)
```

Comment:
Actually: https://github.com/Geod24/ocean/commit/72beac09a1ba4b2374d0abead6d5349ad178f6b3

Reply:
Oh I can remove this part if this commit is to be merged.

Reply:
@Geod24 ping